writer.go

```diff
@@ -0,0 +1,166 @@
+package sequencefile
+
+import (
+	"bytes"
+	"compress/gzip"
+	"crypto/rand"
+	"encoding/binary"
+	"io"
+
+	"github.com/golang/snappy"
+)
+
+type Writer struct {
+	Header        Header
+	writer        io.Writer
+	sinceLastSync int
+	compressor    compressor
+}
+
```

Comment:
Let's add a `Create` method that takes a path, a la, `Open`, and wraps the writer in a `bufio.Writer` (and remembers to flush at the end).